docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -111,87 +149,246 @@ It would help getting a bit more organized if our restaurants directory had some
+
+To create a new Strapi Cloud account:
+
+1. Navigate to the [Strapi Cloud](https://cloud.strapi.io) login page.
+2. Click the **Continue with GitHub** button and log in with the GitHub account where your Strapi project's repository is hosted.
+
+You should now see the Strapi Cloud dashboard. This is where you manage your Strapi projects hosted on Strapi Cloud.
+
+We will create a new Strapi Cloud project by importing the local Strapi project you have just pushed to a GitHub repository:
+
+<ThemedImage
+  alt="Strapi Cloud dashboard"
+  sources={{
+    light: '/img/assets/quick-start-guide/qsg-strapi-cloud-1.png',
+    dark: '/img/assets/quick-start-guide/qsg-strapi-cloud-1_DARK.png',
+  }}
+/>
+
+1. Click the **+ Create project** button.
+2. Select the free trial plan.
+3. Scroll down, and in the "Import git repository section", choose the appropriate Account and Repository from the list (for instance, Account: `yourname`, Repository: `my-first-strapi-project`).
+4. Scroll down further, and in the "Setup" section, give your project a Display name (for instance `my-first-strapi-project`) and leave the other options unchanged.
+5. Click **Create project** at the bottom of the page.
+
+Your Strapi project should be deployed within minutes. 🚀 Once it's done, you'll be able to log into your deployed Strapi project by clicking the **Visit app** button in the top right corner.
+
+</details>
+
+:::callout 🥳 CONGRATULATIONS!  
+Now your project is hosted on Strapi Cloud and accessible online. You can learn more about Strapi Cloud by reading [its dedicated documentation](/cloud/intro) or proceed to part D to log in into your online Strapi project and add your first data from there.
+:::
+
+## 📝 Part D: Add content to your Strapi Cloud project with the Content Manager
+
+Now that we have created a basic data structure with 2 collection types, "Restaurant" and "Category", and deployed your project to Strapi Cloud, let's use the Cloud to actually add content by creating new entries.
+
+<details style={detailsStyle}>
+<summary style={summaryStyle}>Step 1: Log in to the admin panel of your new Strapi Cloud project</summary>
+
+### Step 1: Log in to the admin panel of your new Strapi Cloud project
+
+Now that your Strapi Cloud project is created, let's in into the project:
+
+1. From your [Strapi Cloud dashboard](https://cloud.strapi.io/projects), click the `my-first-strapi-project` project.
+3. Click the **Visit app** button.
+4. In the new page that opens, complete the form to create the first administrator user of this Strapi Cloud project.
+
+<ThemedImage
+alt=""
+sources={{
+  light: '/img/assets/quick-start-guide/qsg-first-login-cloud.gif',
+  dark: '/img/assets/quick-start-guide/qsg-first-login-cloud_DARK.gif'
+}}
+/>
+
+Logged in into our first Strapi Cloud project, we will now add data from there.
+
+<details>
+<summary>ℹ️ Additional information and tips about users and Strapi Cloud projects:</summary>
```

Comment:
If we move the sentence l. 258 above the gif, I'm wondering if it's worth keeping the note & tip callouts below in a toggle? 🤔  I think they already look clean enough, and they're not that long, we could display them by default. What do you think?

Reply:
Hmm, that's interesting, because I found them too long 😅 And I would also say you could totally skip them and successfully follow the rest of the tutorial. I was not even sure I should include this in the guide 🤔 I was afraid we would lose people with too many information, because the guide is already quite long. 

If we keep them, would you say it's better/a best practice (less clicks, maybe?) if we move them out of the details block? 😊 Or would it be OK to leave them as-is?

Reply:
Fair point! I think both options would be fine but I'm totally okay with leaving the callouts in the toggle if you think that's better 😄